pandas/core/generic.py

```diff
@@ -5661,22 +5661,59 @@ def clip_lower(self, threshold, axis=None, inplace=False):
         """
         Return copy of the input with values below given value(s) truncated.
 
+        It truncates values below a certain threshold. Threshold can be a single
+        value or an array, in the latter case it performs the truncation
+        element-wise.
+
         Parameters
         ----------
         threshold : float or array_like
+            Minimum value allowed. All values below threshold will be
+            set to this value.
         axis : int or string axis name, optional
             Align object with threshold along the given axis.
         inplace : boolean, default False
             Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+                .. versionadded:: 0.21.0.
 
         See Also
         --------
-        clip
+        clip : Return copy of input with values below and above thresholds truncated.
```

Comment:
don't we have a clip_upper too, that it's worth listing here? Also, I didn't check, but I understand that the clip listed here is `pandas.clip`, and this is a method of `Series` and `DataFrame`. Is there a `Series.clip` too? We may want to list it here as well then.

Reply:
Yes, there is Series.clip and DataFrame.clip and Series.clip methods. I listed both in the last commit